src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -2,52 +2,66 @@
 
 import com.kodilla.ecommercee.domain.dto.CartDto;
 import com.kodilla.ecommercee.domain.dto.CartItemDto;
-import lombok.RequiredArgsConstructor;
+import com.kodilla.ecommercee.domain.dto.OrderDto;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
-
 import java.math.BigDecimal;
+import java.time.LocalDate;
 import java.util.ArrayList;
 import java.util.List;
 
+
 @RestController
 @CrossOrigin("*")
-@RequestMapping(consumes = MediaType.APPLICATION_JSON_VALUE, value = "/carts")
+@RequestMapping("/carts")
 public class CartController {
     
-    @PostMapping(value = "/createCart/{userId}")
-    public ResponseEntity<Void> createCart(@PathVariable Long userId) {
-        List<CartItemDto> cart = new ArrayList<>();
-        //post to repository//
-        return ResponseEntity.ok().build();
+    @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> createCart(@RequestBody CartDto cartDto) {
+        return new ResponseEntity<>("New cart was created for userid: " + cartDto.getUserId(), HttpStatus.OK);
     }
     
-    @GetMapping(value = "/{userId}/{cartId}")
-    public ResponseEntity<CartDto> getCart(@PathVariable Long userId, @PathVariable Long cartId) {
-        //get from repository//
-        //map to Dto//
-        CartDto currentCard = new CartDto(userId, cartId, new ArrayList<>());
-        return ResponseEntity.ok(currentCard);
+    @GetMapping("/products")
+    public ResponseEntity<List<CartItemDto>> getCart(@RequestParam long cartId) {
```

Comment:
IMHO I will modify this:
    @GetMapping("/cart/{cartId}")
    public ResponseEntity<List<CartItemDto>> getCart(@PathVariable Long cartId) {

Reply:
You might be right. Lets use @PathVariable here.

Reply:
Yes, PathVariable will be better